File: 백준/Bronze/16968. 차량 번호판 1/차량 번호판 1.java
import java.util.*;

public class Main {
    public static void main(String args[]) {
      Scanner s = new Scanner(System.in);
      String car = s.nextLine();
      int[] num = {1,1,1,1};
      for(int i=0;i<car.length();i++)
      {
          if(car.charAt(i) == 'd') num[i] = 10;
          else num[i] = 26;
      }
      for(int i=0;i<car.length()-1;i++)
      {
          if(car.charAt(i) == car.charAt(i+1)) num[i+1]--;
      }
      System.out.println(num[0]*num[1]*num[2]*num[3]);
    }
}

File: 백준/Bronze/16968. 차량 번호판 1/차량 번호판 1.cs
using System;

class Program
{
    static void Main() {
        string car;
        car = Console.ReadLine();
        int[] num = new int[]{1,1,1,1};
        for(int i=0;i<car.Length;i++)
        {
            if(car[i] == 'd') num[i] = 10;
            else num[i] = 26;
        }
        for(int i=0;i<car.Length-1;i++) if(car[i] == car[i+1]) num[i+1] -= 1;
        
        Console.Write(num[0]*num[1]*num[2]*num[3]);
    }
}